doc/manual/manual.bib

```diff
@@ -11306,19 +11308,19 @@ @misc{aspect-doi-v2.0.0
                   others},
   month        = may,
   year         = 2018,
-  doi          = {10.5281/zenodo.1244587},
-  url          = {https://doi.org/10.5281/zenodo.1244587},
-  organization = {Computational Infrastructure for Geodynamics}, 
-  address      = {Davis, CA}, 
+  DOI          = {10.5281/zenodo.1244587},
+  URL          = {https://doi.org/10.5281/zenodo.1244587},
+  organization = {Computational Infrastructure for Geodynamics},
+  address      = {Davis, CA},
 }
 
-@Article{aspectmanual,
-  Title         = {{\textsc{ASPECT}: Advanced Solver for Problems in Earth's ConvecTion, User Manual}},
-  Author        = {W. Bangerth and J. Dannberg and R. Gassm{\"o}ller and T. Heister and others},
-  Year          = {2017},
-  Month         = {4},
+@article{aspectmanual,
+  title         = {{\textsc{ASPECT}: Advanced Solver for Problems in Earth's ConvecTion, User Manual}},
+  author        = {W. Bangerth and J. Dannberg and R. Gassm{\"o}ller and T. Heister and others},
+  year          = {2018},
+  month         = {5},
   DOI           = {10.6084/m9.figshare.4865333},
-  Note          = {doi:10.6084/m9.figshare.4865333},
+  note          = {doi:10.6084/m9.figshare.4865333},
```

Comment:
the article He_2017_DG uses the same old capitalization, care to change that as well?